libc/src/__support/str_to_float.h

```diff
@@ -1055,6 +1056,27 @@ hexadecimal_string_to_float(const char *__restrict src,
   return output;
 }
 
+LIBC_INLINE uint64_t
+nan_mantissa_from_ncharseq(const cpp::string_view ncharseq) {
+  uint64_t nan_mantissa = 0;
+
+  if (ncharseq.data() != nullptr && isdigit(*ncharseq.data())) {
```

Comment:
nit: `*ncharseq.data()` should be `ncharseq[0]`

Reply:
done